### PR TITLE
Change UI to be more similar to current extension: footer

### PR DESCRIPTION
**Pull request by buberdds**
- add fixed menu to footer
- tweak footer css for mobile

Screenshot 2022-07-14 at 15 17 07
Screenshot 2022-07-14 at 15 17 13

### CODE REVIEW

File: src/app/components/MobileFooterNavigation/index.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import { useSelector } from 'react-redux'
+import { useTranslation } from 'react-i18next'
+import { Box, Text } from 'grommet'
+import { LineChart, Money } from 'grommet-icons'
+import styled from 'styled-components'
+import { normalizeColor } from 'grommet/utils'
+import { NavLink } from 'react-router-dom'
+import { selectAddress } from 'app/state/wallet/selectors'
+
+const StyledMobileFooterNavigation = styled(Box)`
```
Comment (lukaw3d) [inline src/app/components/MobileFooterNavigation/index.tsx:11]: `styled(Nav)` semantically better (but may need to readjust gap size)

File: src/app/components/MobileFooterNavigation/index.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+  if (!isMobile || !isAccountOpen) {
+    return null
+  }
+
+  return (
+    <StyledMobileFooterNavigation data-testid="mobile-footer-navigation">
+      {getMenuItems.map(({ label, Icon, to }) => (
+        <NavLink to={to} key={to}>
+          <Box as="span" justify="center" align="center">
```
Comment (lukaw3d) [inline src/app/components/MobileFooterNavigation/index.tsx:60]: Clickable area is a very tight. I'd give it `pad={{ horizontal: 'medium' or '12px' }}`

File: src/app/components/MobileFooterNavigation/__tests__/index.test.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+const renderComponent = (store: any, { isAccountOpen, isMobile }: MobileFooterNavigationProps) =>
+  render(
+    <Provider store={store}>
+      <MemoryRouter>
+        <MobileFooterNavigation isAccountOpen={isAccountOpen} isMobile={isMobile} />
+      </MemoryRouter>
+    </Provider>,
+  )
+
+describe('<isAccountOpen />', () => {
```
Comment (lukaw3d) [inline src/app/components/MobileFooterNavigation/__tests__/index.test.tsx:19]: Typo

**Review by lukaw3d — APPROVED** [review]
Nice!

File: src/app/components/Footer/__tests__/index.test.tsx [hunk 4]
```diff
@@ -19,8 +53,14 @@ describe('<Footer />', () => {
...
-    render(<Footer />)
+    renderComponent(store, 'large')
 
     expect(screen.getByText(/Powered by Oasis Scan API.*/)).toBeInTheDocument()
   })
+
+  it('should render mobile version of footer', () => {
+    renderComponent(store, 'small')
+
+    expect(screen.getByTestId('footer')).toMatchSnapshot()
```
[the remark this answers is not in the document]
Comment (buberdds) [reply src/app/components/Footer/__tests__/index.test.tsx:64]: good idea

**Pull request merged**